README.md

```diff
@@ -5,7 +5,7 @@
 [![Node version](https://img.shields.io/node/v/levelup.svg)](https://www.npmjs.com/package/levelup)
 [![Travis](https://img.shields.io/travis/com/Level/levelup.svg?logo=travis&label=)](https://travis-ci.com/Level/levelup)
```

Comment:
```suggestion
[![Test](https://github.com/Level/levelup/actions/workflows/test.yml/badge.svg)](https://github.com/Level/levelup/actions/workflows/test.yml)
```

I initially forgot this on `concat-iterator` and `supports` which I reckon you used as reference.